### DISCUSSION TITLE
花火厨有没有？

### DISCUSSION THREAD

**Discussion by Liruijia666** [share121/inter-knot#4924] [崩坏：星穹铁道]
![mmexport1726413393524](https://github.com/user-attachments/assets/caabeb4e-e338-42e5-bdaa-a2aa75e9ce3c)

**Comment by JT9364** [comment]
看上媒婆了怎么办（急

**Comment by Liruijia666** [comment]
没事，贫僧可以帮忙超度一下你的

**Comment by Louis-Harsune** [comment]
你猜？（bushi

**Comment by zzz20100624** [comment]
![img-2030773039_1726416371529](https://github.com/user-attachments/assets/e12fb314-31da-481c-90e3-35a0a14b1b88)
我不是

**Comment by tihuahuo** [comment]
乐子神在上，我必须是花火厨

**Comment by Liruijia666** [comment]
必须乐子

**Comment by Fluorine-z** [comment]
![微信图片_20240918205346](https://github.com/user-attachments/assets/b7115b16-4476-4067-8a05-749579a37ce0)
哦~乐子神在上

**Comment by 1cupFor-deadintheafternoon** [comment]
![20240909_233433](https://github.com/user-attachments/assets/623f590c-5dea-4ef2-b802-815e8dbdd6ba)
小灰毛～

**Comment by fzdqz** [comment]
只想超爆不想厨

**Comment by Aha-SWYH** [comment]
必有的

**Comment by wshtdgo** [comment]
必须的！

**Comment by TaiJiDaoZHu** [comment]
花火大人，花火大人！

**Comment by manyshoes** [comment]
pv里有几帧病娇感很足，虽然姿势是我妻由乃的捂脸，但一眼看过去简直就是狂三
